topostats/io.py

```diff
@@ -149,16 +159,100 @@ def _load_jpk(self) -> None:
             LOGGER.error(f"[{self.filename}] : {exception}")
         return jpk
 
+    # uncomment below func and import libasd to use asd formats
+    #def load_asd(self) -> tuple:
+    #    """Extract image and pixel to nm scaling from .asd files.
+
+    #    Returns
+    #    -------
+    #    tuple(np.ndarray, float)
+    #        A tuple containing the image and its pixel to nanometre scaling value.
+    #    """
+    #    try:
+    #        scan = libasd.read_asd(self.img_path)
+    #        print(type(scan))
+    #        LOGGER.info(f"[{self.filename}] : Loaded image from : {self.img_path}")
+    #    except FileNotFoundError:
+    #        LOGGER.info(f"[{self.filename}] File not found : {self.img_path}")
+    #    try:
+    #        # libasd docs seem like there is only 2 channels i.e. help(scan)
+    #        # libasd.Header_v0 uses "type" and _v1 uses "kind"
```

Comment:
Is it possible to determine which file version is being used?

If so it would perhaps be sensible to convert this to an `if: ... else:` with a view to possible changes in the future if/when a `_v2` comes along.

Reply:
Yep, I've added looking for the header types to the file